CONTRIBUTING.md

```diff
@@ -1,43 +1,95 @@
-Steem blockchain, please open an issue in the [Steem Repository](https://github.com/steemit/steem).
-
-## Pull Requests
-
-We love working code, but please do not open a Pull Request unless all the conditions in the **Bumps for working code:**
-section have been met.
 
+# Contributing to Condenser
+
+Everyone is welcome to contribute code to Steemit, Inc. projects. We have a Code of Conduct;
+we ask that you please follow it in all your interactions with our team and your fellow
+contributors.
+
+Please note that this contributing guide only applies to our development presence.
+
+## How to contribute
+
+1. File an issue describing the problem you would like to
+   solve or the feature you would like to add. It saves unnecessary work on your part, 
+   in case there's a nontechnical reason we can't accept a particular change.
+   Please note that we can and will _automatically close all pull requests that do not refer to
+   an issue_.
+2. Learn the conventions of the project you are submitting code to, even implicit ones,
+   and follow them. In case you have a question about a project's conventions, please
+   ask on the issue you filed. This includes writing tests, formatting code, and documentation.
+3. File a pull request against your Github issue and fill out the pull request template.
+   Remember, you are agreeing to license your code and any intellectual property associated
+   with your code under the same terms as the repository you are contributing to.
+4. If one of our teammates asks you to make changes, please make them. Our rule is that all
+   requests for changes in code review are resolved in favor of the reviewer. Our teammates
+   follow this rule too!   
+5. **Reporting Bugs** If there is an existing feature that is not working correctly, or a 
+    glitch in the website that is impacting user behaviour - please open an issue to report 
+    variance. Include as much relevant information as you can, including screen shots and 
+    steps to reproduce the issue.
+6. **Enhancement Suggestions** If you have an enhancement you would like to see in condenser 
+    we suggest you first discuss it with other members of the community on Steemit.com, or 
+    in the steemit discord channel. If there is a well vetted idea that has the support of 
+    the community create an issue with a link to the discussion, a detailed description, and 
+    tag it with the label `community-src`
+7. **Development Discussions** The developers frequently open issues to discuss changes that 
+    are being worked on. This is to inform the community of the changes being worked on, and 
+    to get input from the community and other developers on the implementation.    
+8. **Pull Requests** We love community contributions. If you've done some work on an issue
+     and want to see it in the condenser codebase, please submit a Pull Request, reference
+     the issue in the description *and tag the it with `community-src`* we're working very
```

Comment:
I don't think that as community contributors we have the ability to add tags to an issue (or PR). Can you clarify how we should handle this part?

Reply:
@TimCliff - please take a look.